components/typography/typography.scss

```diff
@@ -137,3 +137,26 @@
 	font-weight: 700;
 	letter-spacing: 0.2px;
 }
+
+
+@mixin d2l-blockquote {
```

Comment:
Debatable whether we need to keep supporting Sass with new stuff I think? I'd love to be able to phase it out one day when everything is Lit, so the more we add here the harder that'll be.

Reply:
If you want I can exclude the blockquote stuff. I just added it in all files.

Reply:
Yeah I would exclude it.